### DISCUSSION TITLE
Welcome to airunner Discussions!

### DISCUSSION THREAD

**Discussion by w4ffl35** [Capsize-Games/airunner#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by convinced-prog** [comment]
downloaded airunner 3gb and bitdefender flags it as a triojan. Also what are all the 'geographic' files for

**Reply by w4ffl35** [reply]
Hi - it seems that you are getting a false positive. 
I have scanned with Norton Antivirus and Bitdefender and both come up clean. Additionally I have tested with Microsoft Defender and it hasn't flagged anything.

All of the files in airunner are binaries gathered from installed python packages - the locale files you're seeing are binaries gathered from one of the required python libraries.

---

### Additional information about the build process

Installed libraries can be viewed here:

https://github.com/Capsize-Games/aihandler/blob/develop-windows/setup.py

Builds are handled in docker containers which are stored in Github.

Pyinstaller is used to compile AI Runner (this happens in the Docker container mentioned above).

[This spec file is used by pyinstaller for builds](https://github.com/Capsize-Games/airunner/blob/master/build.airunner.windows.prod.spec) and can be viewed to gain additional insight into which libraries are used in the final build.

---

### Scan results of Norton Antivirus

```
Scan Statistics:
  Scan Start:
   Local: 4/14/2023 3:08 PM
   UTC: 4/14/2023 9:08 PM
  Scan Time: 0 seconds
  Scan Targets: D:\ai-runner\airunner.exe
  Counts:
   Total items scanned: 1
   - Files & Directories: 1
   - Registry Entries: 0
   - Processes & Startup Items: 0
   - Network & Browser Items: 0
   - Other: 0
   - Trusted Files: 0
   - Skipped Files: 0

   Total security risks detected: 0
   Total items resolved: 0
   Total items that require attention: 0

Resolved Threats:
No risks have been resolved

Unresolved Threats:
No unresolved risks
```

### Scan results for bitdefender

[AIRunner-Bitdefender_scan_results.txt](https://github.com/Capsize-Games/airunner/files/11237255/AIRunner-Bitdefender_scan_results.txt)

**Comment by w4ffl35** [comment]
Let me know if you still run into the issue. If so, more details will help (for example if you're taking a particular action and the alert triggers).

**Comment by convinced-prog** [comment]
Hi
Thanks for the info
It's running now so no problem.
While I'm here, can I add other models that I have to the program or do I
have to download them as needed.
Thanks again

**Reply by w4ffl35** [reply]
[Take a look at the wiki for information on using your own models](https://github.com/Capsize-Games/airunner/wiki/AI-Runner-Manual)

Also, FYI the timezone files you asked about are coming from `tcl\tcl8.6\tzdata\` which is a built in python library (no cause for concern).

**Comment by w4ffl35** [comment]
Closing this discussion as it is not relevant

1. The software obviously isn't a trojan - all code is on display here along with docker image and a full view into what is being compiled
2. No longer being built with timezone files.
